lang/rust/Makefile

```diff
@@ -0,0 +1,138 @@
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=rust
+PKG_VERSION:=1.49.0
+PKG_RELEASE:=1
+PKG_LICENSE:=Apache-2.0 MIT
+PKG_LICENSE_FILES:=LICENSE-APACHE LICENSE-MIT
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=https://github.com/rust-lang/rust.git
+PKG_SOURCE_DATE:=2020-11-11
+#PKG_SOURCE_VERSION:=b2d115f6db5172c961dfeb50de15f35784dbc7c9
+#PKG_SOURCE_VERSION:=5a6a41e7847ff5f85a31b87431ce2af29c567f1d
+PKG_SOURCE_VERSION:=fa55f668e5ea5388ec98b9340969527252239151
+PKG_HASH:=skip
```

Comment:
Why are you trying to skip hash at all?
Take a look here https://openwrt.org/docs/guide-developer/packages#buildpackage_variables and there is how you can check hash for the tarball.

Reply:
For the same reason I have multiple PKG_SOURCE_VERSION.  These are "milestones" in the constantly shifting Rust Source code.  I was actively working with the (and Suricata devs) to get this to work with OpenWrt and it required/requires a LOT of going back and forth for testing.  I'm essentially trying to prove Rust works by making Suricata work.  Giving me the ability to quickly jump between milestones was/is something I need to hold on to for the moment, but will be removed and SHA256 hashed correctly before committing to the upstream.

Reply:
> Giving me the ability to quickly jump between milestones was/is something I need to hold on to for the moment, but will be removed and SHA256 hashed correctly before committing to the upstream.

Why you don't want to have CI passed in the meantime, so anyone can download artifacts from this PR?
Right now, nobody knows that it even compiles.
See: https://github.com/openwrt/packages/runs/1548152198?check_suite_focus=true#step:4:3730